README.md

```diff
@@ -1,77 +1,77 @@
 ## Firefox Application Services
 
-_A platform for building cloud-powered applications that target Firefox users._
+Application Services (a-s) is collection of Rust Components that are used to enable Firefox applications to integrate with Firefox accounts, sync, experimentation, etc. Each component is built using a core of shared code written in Rust, wrapped with native language bindings for different platforms.
 
-### What's this all about?
+### Contributing
+To contribute, please review the Mozilla [Community Participation Guidelines](https://www.mozilla.org/en-US/about/governance/policies/participation/) and then visit our [how to contribute](docs/contributing.md) guide.
```

Comment:
It's somewhat confusing to send folks to contributing.md, because that page covers some of the same stuff as `building.md`, but proposes using `cargo asdev` to do so.  

The `cargo asdev` as-dev instructions are much simpler, but I'm guessing there are reasons we're not using it here too?

In any case, the solution to the duplicated instructions is probably just to remove all build-environment-related instructions from `docs/contributing.md` and link to `docs/building.md`.

Reply:
I intend to update the Contributing file as well - just not as part of this set of changes.